.drone/cmd/ghcomment/main.go

```diff
@@ -0,0 +1,111 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"io/ioutil"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/shurcooL/githubv4"
+
+	"github.com/grafana/influx2cortex/drone/pkg/github"
+)
+
+const CommenterLogin = "grafanabot"
+
+func main() {
+	grafanabotPat := getRequiredEnv("GRAFANABOT_PAT")
+
+	repoOwner := getRequiredEnv("DRONE_REPO_OWNER")
+	repoName := getRequiredEnv("DRONE_REPO_NAME")
+
+	pullRequest, err := strconv.Atoi(getRequiredEnv("DRONE_PULL_REQUEST"))
+	if err != nil {
+		panic(err)
+	}
+
+	commentTypeIdentifier := flag.String("i", "", "String that identifies the comment type being submitted")
+	commentBodyFilename := flag.String("b", "", "A file containing the comment body")
+	flag.Parse()
+
+	if *commentTypeIdentifier == "" {
+		panic("Required argument: -i")
+	}
+	if *commentBodyFilename == "" {
+		panic("Required argument: -b")
+	}
+
+	api := github.NewAPI(context.Background(), grafanabotPat)
+
+	err = minimizeOutdatedComments(api, repoOwner, repoName, pullRequest, *commentTypeIdentifier)
+	if err != nil {
+		panic(err)
+	}
+
+	commentBody, err := ioutil.ReadFile(*commentBodyFilename)
+	if err != nil {
+		panic(err)
+	}
+
+	err = addComment(api, repoOwner, repoName, pullRequest, string(commentBody))
+	if err != nil {
+		panic(err)
+	}
+}
+
+func getRequiredEnv(k string) string {
+	v, p := os.LookupEnv(k)
+	if !p {
+		panic("Missing required env var: " + k)
+	}
+
+	return v
+}
+
+func minimizeOutdatedComments(
```

Comment:
from googling around I think the goland formatting is a bit non-standard.  If it seems to insist on this formatting, there seems to be a way to force it to use gofmt instead: https://stackoverflow.com/questions/47735678/goland-how-to-use-gofmt

I'm mostly trying to avoid a formatter war where prs keep reformatting the same code over and over